### PR TITLE
Squash profile and user edit page into one

### PR DESCRIPTION
**Pull request by cmfcmf**
Fixes https://github.com/hpi-swt2/workshop-portal/issues/245 `US_2.44: Profile/User-Settings`

### CODE REVIEW

File: app/controllers/users/registrations_controller.rb [hunk 1]
```diff
@@ -44,9 +44,15 @@ class Users::RegistrationsController < Devise::RegistrationsController
   # end
 
   # If you have extra params to permit, append them to the sanitizer.
-  # def configure_account_update_params
-  #   devise_parameter_sanitizer.permit(:account_update, keys: [:attribute])
-  # end
+  def configure_account_update_params
+    devise_parameter_sanitizer.permit(:account_update) do |user|
+      user.permit(:email, :password, :password_confirmation, :current_password, {:profile => [:first_name, :last_name, :gender, :birth_date, :school, :street_name, :zip_code, :city, :state, :country, :graduates_school_in]})
```
Comment (cmfcmf) [inline app/controllers/users/registrations_controller.rb]: This line always triggers `undefined method `concat' for #<Proc:0x007f65c4439870>`  after I try to save changes. I tried several different things and this is where I'm stuck.

File: app/controllers/users/registrations_controller.rb [hunk 2]
```diff
@@ -44,9 +44,13 @@ class Users::RegistrationsController < Devise::RegistrationsController
   # end
 
   # If you have extra params to permit, append them to the sanitizer.
-  # def configure_account_update_params
-  #   devise_parameter_sanitizer.permit(:account_update, keys: [:attribute])
-  # end
+  def configure_account_update_params
+    devise_parameter_sanitizer.permit(:account_update, keys: [:profile => [:first_name, :last_name, :gender, :birth_date, :school, :street_name, :zip_code, :city, :state, :country, :graduates_school_in]])
```
Comment (cmfcmf) [inline app/controllers/users/registrations_controller.rb]: This is the line that produces all the issues :/

**Comment by cmfcmf** [comment]
This is finally done @tom95 @hendraet

**Review by nstrelow — APPROVED** [review]
Looks gut 2 me

**Pull request merged**